src/modules/ethereum/account-abstraction/rewards.ts

```diff
@@ -0,0 +1,33 @@
+import { FEATURE_LOYALTY_FLOW } from 'src/env/config';
+import { useFirebaseConfig } from 'src/modules/remote-config/plugins/useFirebaseConfig';
+
+export interface GasbackData {
+  /** fixed gasback for a particular trade */
+  value?: number;
+  /** potential gasback for a trade */
+  estimation?: number;
+}
+export function useGasbackEstimation({
+  paymasterEligible,
+  suppportsSimulations,
+  supportsSponsoredTransactions,
+}: {
+  paymasterEligible: null | boolean;
+  suppportsSimulations: boolean;
+  supportsSponsoredTransactions: boolean | undefined;
+}) {
+  const { data, ...query } = useFirebaseConfig(['loyalty_config'], {
+    enabled:
+      FEATURE_LOYALTY_FLOW === 'on' &&
```

Comment:
Hah, I see that we can just leave `FEATURE_LOYALTY_FLOW && ...`

Reply:
Yeah, I think so..., because we have this line in `config.ts`:
`process.env.FEATURE_LOYALTY_FLOW === 'on' ? 'on' : null`

Reply:
Yes but I think what I tried here is to avoid false-positives by doing two things:
1. converting value other than `'on'` to `null` early on (maybe it should be `false`)?
2. AND have a habit to explicitly check for `'on'` in code.

That way if one part fails the result would still be correct :) 

Is this overkill?